docs/api/commands/screenshot.mdx

```diff
@@ -300,6 +300,17 @@ cy.screenshot()
 cy.get('.sticky-header').invoke('css', 'position', null)
 ```
 
+### Chromium-specific behavior with regard to tabs
+
+Chromium will not capture screenshots when the Rendeer process for the Cypress
```

Comment:
Is this supposed to say "Render" process or `RenderProcess` perhaps?

https://www.chromium.org/developers/design-documents/multi-process-architecture/

Reply:
Renderer, actually - thank you for the catch!